data/DataType/boolean_title_case.yaml

```diff
@@ -0,0 +1,6 @@
+name: boolean
+integer-min: '0'
+integer-max: '1'
+literals:
+  - 'True'
+  - 'False'
```

Comment:
Please change this to `TRUE` and `FALSE`. ([ref](https://ideone.com/Taf3Lr)) Create a new file named `bool_upper_case.yaml` (if not present already) and put those there.

Reply:
:+1:

Reply:
Okay